fix(branding): update "Ubuntu Advantage" to "Ubuntu Pro" #6276

sowasred2012 opened the pull request:
## Done

- updated "Ubuntu Advantage" copy to "Ubuntu Pro" (and one instance of "Ubuntu Advantage for Infrastructure" to "Ubuntu Pro (Infra-only)")

## QA

- Check out this branch
- Search the codebase for "Ubuntu Advantage", there should be no results

Fixes https://github.com/canonical/web-squad/issues/6276

Lukewh commented:
@sowasred2012  The only place I can find "Advantage" is in test cassettes. I imagine that's is why the python tests are failing?

sowasred2012 commented:
@Lukewh good spot, I've updated anyway but seems the test-python action is having issues with the version of Python - on @jkfran suggestion, I've updated that 20.04, which uses Python 3.8 and has talisker

Lukewh reviewed (approved):
:+1: LGTM
